Module/main.js

```diff
@@ -125,12 +140,18 @@ class BugReportForm extends FormApplication {
 
     const fullDescription = [[issuerString, labelString].join('\n'), versions.join('\n'), descriptionString].join('\n \n');
 
+    let bugsUrl = this.endpoints.bugs;
+    // construct gitlab link (if applicable)
+    if (this.gitlab) {
+      bugsUrl = bugsUrl + `?title=${encodeURIComponent(bugTitle)}&labels=${encodeURIComponent(label)}&description=${encodeURIComponent(fullDescription)}`;
```

Comment:
not sure the `labels` is right for us to include as our 'labels' might not match up with the repo's labels